src/pages/Home.tsx

```diff
@@ -0,0 +1,150 @@
+                <div className="home__intro__bubbles bubbles--4" />
+                <div className="home__intro__bubbles bubbles--5" />
+                <div className="home__intro__bubbles bubbles--6" />
+                <div className="home__intro__bubbles bubbles--7" />
+                <div className="home__intro__bubbles bubbles--8" />
+                <div className="home__intro__bubbles bubbles--9" />
+                <img src="/images/logo.png" alt="" className="home__intro__logo" />
+                <div className="home__intro__container">
+                    <div className="home__intro__container__image" />
+                    <p className="home__intro__container__info">ROBERTA <br /> 1v1.0</p>
+                </div>
+                <img src="/images/turtle1.png" alt="" className="home__intro__turtle"/>
+            </div>
+        
+            <div className= "home__about">
+                <p className="home__about__title">About WiE Design Team</p>
+                <p className="home__about__body">
+                    Women in Engineering Design Team (WiE Design) is a small technical club
+                    that is a subset of the larger Women in Engineering club at Simon Fraser
+                    University. WiE Design consists of self-identifying females and other
+                    under-represented groups in STEM. We have members from a wide range of
+                    programs such as Computer engineering, Systems engineering, Electronics
+                    engineering, Bio-med engineering, Mechatronics engineering, Communications
+                    and Business. Our goal is to give women a safe and uninterrupted platform
+                    to share, and listen to ideas from other women, build their design knowledge
+                    and technical skills in fields such as engineering, marketing and business,
+                    and most importantly to gain confidence in their abilities.
+                </p>
+            </div>
+
+            <div className="home__robot">
+                <img className="home__robot__turtle" src="/images/turtle2.png" alt="" />
+                <div className="home__robot__top">
+                    <div className="home__robot__top__bubbles bubble--1" />
+                    <div className="home__robot__top__bubbles bubble--2" />
+                    <div className="home__robot__top__bubbles bubble--3" />
+                </div>
+                <p className="home__robot__text">Meet<br/>Our<br/>Robot</p>
+                <div className="home__robot__bottom">
+                    <div className="home__robot__bottom__bubbles bubble--4" />
+                    <div className="home__robot__bottom__bubbles bubble--5" />
+                    <div className="home__robot__bottom__bubbles bubble--6" />
+                </div>
+                <div className="home__robot__model">
+                    <model-viewer
+                        src="/3d-model/SN_RobotAssembly.glb"
```

Comment:
Just a suggestion, I think it this would look nicer if it was an import (ie. `import ROBERTAModel from "/3d-model/SN_RobotAssembly.glb"`). Then you can just pass in `ROBERTAModel` to this component.

Reply:
If you want, you could also import the images too.

Reply:
I don't think I can import the images since they're in the Public folder and React doesn't let code in the SRC folder import from the Public folder "( @kaypnma